go/iot/arduino/protocol.txt

```diff
@@ -31,9 +34,18 @@ When the program starts, the following view would be registered.
         TagKeys: ["DeviceId", "SampleDate"]
 	ReportPeriod : 1
 
-Before the registration, the Raspberry Pi would make a negative response if there is any record request from Arduino.
+Before the registration, the Raspberry Pi would not make any response to the the request from Arduino.
+
+After the registration, the Raspberry Pi would make a negative or positive response to any record request from Arduino.
+
+When the Pi would make a negative response:
+    1. The measure doesn't bind to any views.
+    2. The tag key doesn't bind to any views.
```

Comment:
Why would this be a problem? If there's an unknown tag key, you can simply drop it.